source/user-manual/registering/password-authorization-registration.rst

```diff
@@ -7,11 +7,11 @@
 Registration service with password authorization
 ================================================
 
-This method is similar to the :ref:`simple registration service <simple-registration-service>`, except that it provides additional protection of the Wazuh manager from unauthorized registrations by using a password.
+This method is similar to the :ref:`simple registration service <simple-registration-service>`, except that it provides protection from unauthorized registrations by requiring a password during the registration process.
 
-Before the registration process, enabling the password authorization option and creating the registration password has to be done on the Wazuh manager. This password can be used for the subsequent agent registrations with the same Wazuh manager.
+Password protected registration is enabled in the Wazuh manager configuration. To protect a multinode cluster from unauthorized agent registrations, password authorization must be enabled on every one of the manager nodes to avoid unauthorized registrations through one of them. The password to be used with agents registration can be the one provided by the service or a custom one defined during the manager configuration.
```

Comment:
```suggestion
Password protected registration is enabled in the Wazuh manager configuration. To protect a multi-node cluster from unauthorized agent registrations, password authorization must be enabled on every one of the Wazuh manager nodes to avoid unauthorized registrations through one of them.
```